.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI Pipeline
+
+on:
+  push:
+    branches:
+      - Workflow
+  pull_request:
+    branches:
+      - Workflow
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run ESLint
```

Comment:
You do not need to run eslint and prettier here. Instead, you have ran the e2e and unit test.